Livecheckables/ledger.rb

```diff
@@ -0,0 +1,6 @@
+class Ledger
+  livecheck do
+    url :stable
```

Comment:
```suggestion
    url :head
```

The `ledger` formula has the GitHub repo as `head`, so we can just use that directly.